include/sparrow/variable_size_binary_layout.hpp

```diff
@@ -73,6 +73,42 @@ namespace sparrow
         friend class iterator_access;
     };
 
+    /**
+     * @class vs_binary_reference
```

Comment:
Is this supposed to be used in other layouts? If yes, maybe rename?

Reply:
It's not. The implementation is specific to the variable size binary layout.